packages/playwright-crawler/src/internals/playwright-crawler.ts

```diff
@@ -250,4 +253,18 @@ export class PlaywrightCrawler extends BrowserCrawler<{ browserPlugins: [Playwri
 
         return gotoExtended(crawlingContext.page, crawlingContext.request, gotoOptions);
     }
+
+    protected override async _applyCookies({ session, request, page }: PlaywrightCrawlContext, preHooksCookies: string, postHooksCookies: string) {
+        const sessionCookie = session?.getPuppeteerCookies(request.url) ?? [];
```

Comment:
maybe we should rename the `getPuppeteerCookies` method? also looking at the code, we depend on puppeteer types in core because of it, we will need to fix that too

(let's not deal with that in this PR, just a thought)

Reply:
Opened an issue for this so we can track it for the future 👍